packages/module/src/Message/Message.tsx

```diff
@@ -203,6 +203,8 @@ export interface MessageProps extends Omit<HTMLProps<HTMLDivElement>, 'role'> {
   remarkGfmProps?: Options;
   /** Props for a tool call message */
   toolCall?: ToolCallProps;
+  /** Whether user messages default to stripping out images in markdown */
+  hasNoImagesInUserMessages?: boolean;
```

Comment:
This is just a nit, but wdyt about having this prop be the inverse and be something like "areImagesAllowed" or "hasImages"? Not totally sure if we'd need to note this is for user messages in the prop name, but just to try and make the prop name a tad more concise.